addons/crm/views/crm_team_views.xml

```diff
@@ -192,6 +180,17 @@
             </field>
         </record>
 
+       <record id="crm_team_view_tree" model="ir.ui.view">
```

Comment:
Linting: could you define it before ``sales_team_form_view_in_crm`` ? That way we try to group views definitions.